packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextShadowNode.java

```diff
@@ -75,6 +75,7 @@ public long measure(
                   "Spannable element has not been prepared in onBeforeLayout");
 
           Layout layout = measureSpannedText(text, width, widthMode);
+          TextPaint paint = new TextPaint(sTextPaintInstance);
```

Comment:
Can you amend all the changes to the paper implementation. This is quite intrusive and we'd rather just have the fix for Fabric only.

cc @NickGerleman

Reply:
Done - https://github.com/facebook/react-native/pull/47082/commits/ec2182b7a72e4d77d016fb672b9f8176e34e2bd5